tests/suites/cloud/suite.js

```diff
@@ -301,21 +304,6 @@ module.exports = {
       await this.worker.archiveLogs(this.id, `${this.balena.uuid.slice(0, 7)}.local`,);
     });
 
-    await this.utils.waitUntil(async() => {
-      console.log("Waiting for device to be online...");
-      return await this.cloud.balena.models.device.isOnline(this.balena.uuid);
-    }, false);
-
-    this.log("Device is online and provisioned successfully");
-    
-    await this.utils.waitUntil(async () => {
-      this.log("Trying to ssh into device...");
-      return await this.cloud.executeCommandInHostOS(
-        "cat /etc/hostname",
-        this.balena.uuid
-      ) === this.balena.uuid.slice(0, 7);
-    }, false);
-
```

Comment:
Were these just redundant checks that we don't need?

Reply:
Basically they were yeah. They are checks to see if the device is online in the dashboard essentially, which happens just after this in the preload test anyway